README.md

```diff
@@ -0,0 +1,34 @@
+![github_banner](http://res.cloudinary.com/chi6rag/image/upload/v1431578923/HourGlass/hourglass_banner.png)
+
+HourGlass
+=========
+
+HourGlass lets you track the time you spend physically at a location.
+
+####Application Workflow (Version 0)
+
+######Authenticated User
+- Logs in
+- For one particular day ( from 00:00 hrs to 23:59 hrs )
```

Comment:
As discussed over email, it would be good to support multiple checkins in a day.
Care has to be taken to prevent overlapping checkins!